tests/arch/arm/arm_custom_interrupt/README.txt

```diff
@@ -0,0 +1,58 @@
+Title: Test to verify custom interrupt controller handling with on
+       the Cortex-M architecture.
+
+Description:
+
+This test verifies customer interrupt controller handling on
+Cortex-M architectures using the CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER
+option. It will locate a unused interrupt to trigger via software
+and test all methods that are part of the SoC interrupt control
+interface.
```

Comment:
Please convert this file to restructured text for consistent rendering.

Reply:
Sorry for my ignorance, but could you provide some guidance here? I'm following the template established by other test readmes.

Reply:
My bad, sorry. I didn't realize there was a different convention used in most of the test projects.